deployments/liqo/crds/discovery.liqo.io_peeringrequests.yaml

```diff
@@ -22,14 +22,10 @@ spec:
         description: PeeringRequest is the Schema for the PeeringRequests API
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation
-              of an object. Servers should convert recognized schemas to the latest
-              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
Hi Lorenzo, are you using the `manifests` target of the **Makefile** in Liqo's repo when generating the CRDs? It seems that the version of your `controller-gen` is different that the `0.4.0` used by the makefile. While updating the `clusterconfigs` CRD happened to update all the other CRDs. Can you please revert this changes, possibly using `controller-gen v0.4.0`?